src/System.Management.Automation/engine/remoting/client/JobManager.cs

```diff
@@ -81,51 +81,43 @@ internal void RegisterJobSourceAdapter(Type jobSourceAdapterType)
             Dbg.Assert(jobSourceAdapterType != null, "JobSourceAdapterType should never be called with null value.");
             object instance = null;
 
-            if (jobSourceAdapterType.FullName != null && jobSourceAdapterType.FullName.EndsWith("WorkflowJobSourceAdapter", StringComparison.OrdinalIgnoreCase))
+            ConstructorInfo constructor = jobSourceAdapterType.GetConstructor(Type.EmptyTypes);
+            if (!constructor.IsPublic)
             {
-                MethodInfo method = jobSourceAdapterType.GetMethod("GetInstance");
-                instance = method.Invoke(null, null);
+                string message = string.Format(CultureInfo.CurrentCulture,
```

Comment:
Has the JobManagerRegistrationConstructorError message been removed?

Reply:
No. It's defined in `src/System.Management.Automation/resources/RemotingErrorIdStrings.resx`.

All this PR does is remove the `if` since it no longer applies in the `if/else` statement.